_posts/2020-04-03-rendering-in-UE4.md

```diff
@@ -0,0 +1,32 @@
+---
+author: "alepaoletti"
+layout: post
+did: "blog10"
+title: "Rendering in Unreal Engine 4"
+slug: "RenderingInUE4"
+date: 2020-04-06 08:00:00
+categories: rendering
+img: /banners/GPU_profiler.png
```

Comment:
Follow the example of the semgan article: https://raw.githubusercontent.com/zurutech/zurutech.github.io/master/_posts/2019-07-01-semgan.md

As you can see, the `img` and `banner` fields, contains **relative** paths (`semgan/` and **not** `/semgan`).
The path is relative to the `/images` folder.

Reply:
Got it.